### ISSUE TITLE
changing the app name independently of the language settings

### ISSUE THREAD

**Issue by Leuconoe** [microsoft/terminal#17924]
# Description of the new feature/enhancement

![image](https://github.com/user-attachments/assets/07d610cb-6c0a-4550-b980-a160a1785cb2)
![image](https://github.com/user-attachments/assets/d8cb8423-93a4-4f75-aa16-a30c9c91291e)

For some reason, when I search for “terminal” in the Start menu, it doesn't find Terminal.
My Windows language setting is Korean, and I have to search for “터미널” to find the app.

I know there is a way to create a shortcut icon and add it to the search indexer, but I don't think that's a good idea.

Is there a way to just rename the terminal app regardless of language?

# Proposed technical implementation details (optional)

**Comment by lhecker** [comment]
I don't believe there's a way for us to make the package name configurable. But I also I think that's something that the search indexer should simply do correctly. There's no reason why it wouldn't find Windows Terminal when searching for "terminal".

**Comment by lhecker** [comment]
I asked around and we (the terminal team) apparently talked to the start menu search team years ago and asked them to include "terminal" (in English) as a keyword for Windows Terminal. It _should_ be working. We asked them in the past to ensure it works and at this point there's nothing we can do about this anymore from our side. As such we must close the issue, as we've done all we can. 😟

**Comment by Leuconoe** [comment]
Follow-up:Suddenly it's working again!
